Plots/Satellites/NCL_sat_1.py

```diff
@@ -32,11 +33,31 @@
 U = ds.slp[24, :, :]
 
 # Translate short values to float values
-U = U.astype('float64')
+#U = U.astype('float64')
 
 # Convert Pa to hPa data
 U = U*0.01
 
+def findLowPressureCoords(level):
+
+    # Same as xarray data
+    lat = np.arange(90,-92.5,-2.5) 
+    lon = np.arange(0,360.0,2.5)
+
+    coordlist = []
+
+    for x in range(len(U.data)):
+        for y in range(len(U.data[x])):
+            if (int)(U.data[x][y]) == level:
+                latval = -lat[x]
+                lonval = -lon[y]+270
+                # If in scope of area of map depicted on plot
+                if (0<latval<90) and (-180<lonval<0):
```

Comment:
Instead of limiting this function by the region you are trying to draw a low pressure marker, why not define a low pressure point by how different it is from the pressure at a point nearby? For example, say a local minimum has been found. Check if that minimum has a pressure that is lower than it's surroundings by at least 30 hPa. If so, draw the label. Now the pressure threshold and the distance away for the reference pressure seem arbitrary, but it's a start and could be applied globally.